src/components/sidebar/ConnectedSidebar.js

```diff
@@ -0,0 +1,34 @@
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react-redux'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->